minimesos/src/main/java/com/containersol/minimesos/mesos/ClusterArchitecture.java

```diff
@@ -200,13 +204,17 @@ public Builder withRegistrator(Registrator registrator) {
 
         /**
          * Includes the default {@link MesosMaster} instance in the cluster
+         *
+         * @return TODO(mwl)
          */
         public Builder withMaster() {
             return withMaster(MesosMasterContainer::new);
         }
 
         /**
          * Includes the default {@link MesosAgent} instance in the cluster
+         *
+         * @return TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)